middlewares/error-middleware.js

```diff
@@ -0,0 +1,21 @@
+function errorMiddleware(error, req, res, next) {
+    let code = 500;
+    let message = 'Internal Server Error';
+
+    if (error.name === 'SequelizeUniqueConstraintError') {
+        res.status(400).json({ message: 'Bad Request' });
```

Comment:
in class tomorrow please give all of this example case for error handling

Reply:
noted sir